pages/workflow/implement.mdx

```diff
@@ -0,0 +1,40 @@
+---
```

Comment:
Do we have any opportunities to add screenshots or videos here, to make this page pop a bit?

Inspo, the Avo Functions GIF we have in this deck: https://docs.google.com/presentation/d/1KsyBWXSS2g3AJrETL4-D0ZEdofZtn7IUipPXx3j7VCw/edit#slide=id.g1369a7f867c_0_1036